### DISCUSSION TITLE
parsing syslog from firewall

### DISCUSSION THREAD

**Discussion by DinhDiesel** [Security-Onion-Solutions/securityonion#5774] [Unsupported Versions] [answered]
Question: I have my firewall parsing syslogs to my manager (in distributed environment). I read that in #5470  an edit was made to /opt/so/saltstack/local/pillar/minions/hostname.sls adding this; 

firewall:
  assigned_hostgroups:
    chain:
      DOCKER-USER:
        hostgroups:
          log_sources:
            portgroups:
              - portgroups.syslog
      INPUT:
        hostgroups:
          log_sources:
            portgroups:
              - portgroups.syslog

I recently copied and pasted this to the bottom of my manager.sls, is it ok to point syslogs to manager and i feel like the formatting is not correct.
![tempsnip](https://user-images.githubusercontent.com/87437722/136036517-1eed4781-c737-41df-9056-e42fcfbd1f51.png)

**Comment by dougburks** [comment, marked as answer]
Are you just trying to send standard syslog over standard port 514? If so, have you tried `so-allow`?
https://docs.securityonion.net/en/2.3/so-allow.html

**Reply by DinhDiesel** [reply]
@dougburks  yes I have tried so-allow on the standard port 514, maybe my filtering techniques are off? I tried "destination.port: 514" trying to comb through but i do not see much. I also confirmed from firewall that logs are parsed to manager and the standard port...

**Reply by dougburks** [reply]
Try using `tcpdump` to verify that syslog traffic is actually arriving on port 514.

**Reply by DinhDiesel** [reply]
I did do a tcpdump, I still see a connection attempt but I cant see the actual syslogs... heres a screenshot if this helps?
![noreply](https://user-images.githubusercontent.com/87437722/137925534-f00ac585-0e10-41d0-86f7-8de05a4c9a10.JPG)

**Reply by dougburks** [reply]
When you did a tcpdump, which interface did you sniff from? The monitoring interface or the management interface?

**Reply by DinhDiesel** [reply]
@dougburks I did it from the managersearch node on just the management, my heavy is the one that has the port mirrored (from switch into the sniffer port on heavy)

**Reply by dougburks** [reply]
From your other discussions, it seems like you had just a manager node and not a managersearch and if that's the case your manager has nowhere to store the syslog:
https://github.com/Security-Onion-Solutions/securityonion/discussions/5947#discussioncomment-1513851

**Comment by DinhDiesel** [comment]
also forgot to mention i have the port on my switch that is mirror traffic in the 2nd NIC on my heavy which is in prom mode if that helps anything?
